README.md

```diff
@@ -249,7 +249,7 @@ FetchContent_Declare(
     GIT_TAG        master # or use release tag, eg. v4.0
 )
 # Set this to ON if you want to have the CppUTests in your project as well.
-set(TESTS OFF CACHE BOOL "Switch off CppUTest Test build")
+set(CPPUTEST_BUILD_TESTING OFF CACHE BOOL "Switch off CppUTest Test build")
```

Comment:
This is disabled by default no when used in other projects. I think we can delete this entirely.

Reply:
https://github.com/cpputest/cpputest/blob/f8b2b8c28c1fe28c7558256cf7be083b79d82ff7/CMakeLists.txt#L42-L43

Reply:
Sounds good, removed it